.npmrc

```diff
@@ -3,3 +3,13 @@
 update-notifier=false
 # ESLint editor integrations expect ESLint's binary to be in the root node_modules.
 public-hoist-pattern[]=eslint
+# pnpm v10.3.0+ ships with node-gyp@11.1.0. That version of node-gyp has a bug which prevents
+# node-pty, one of Connect deps, from being built on Windows (https://github.com/nodejs/node-gyp/issues/3126).
+#
+# To work around this, we install node-gyp@11.0.0 as a dev dep and then tell node-pty through the below
+# config option to use node-gyp from our node_modules rather than the one bundled with pnpm.
+#
+# During pnpm install, the script is executed from a directory such as
+# <repo root>/node_modules/.pnpm/node-pty@1.1.0-beta14/node_modules/node-pty
+# so we escape five directories and then give a path to node-gyp.js.
+node_gyp=../../../../../node_modules/node-gyp/bin/node-gyp.js
```

Comment:
I think this part might've been unnecessary. According to https://github.com/microsoft/node-pty/issues/777, it should be enough to add node-gyp to package.json. From my quick tests, it seems to work when running `pnpm rebuild -r node-pty` and `pnpm package-term`.